lib/widgets/branch.dart

```diff
@@ -0,0 +1,363 @@
+                    Task newTask = Task(text);
+                    setState(() {
+                      allTasks.add(newTask);
+                    });
+                    Navigator.of(context).pop();
+                  }
+                },
+              )
+            ]),
+          ],
+        );
+      },
+    );
+  }
+
+  @override
+  Widget _TaskList(List<Task> tasks) {
+    if (tasks.length != 0) {
+      return ListView.builder(
+          itemCount: tasks.length,
+          itemBuilder: (context, index) {
+            return Padding(
+                padding: EdgeInsets.all(6.0),
+                child: Dismissible(
+                    background: Container(
+                      color: Colors.red,
+                      child: Icon(Icons.delete_forever),
+                    ),
+                    key: ValueKey<Task>(tasks[index]),
+                    onDismissed: (DismissDirection direction) {
+                      setState(() {
+                        allTasks.remove(tasks[index]);
+                      });
+                    },
+                    direction: DismissDirection.endToStart,
+                    child: CheckboxListTile(
+                        contentPadding: EdgeInsets.symmetric(horizontal: 8.0),
+                        checkboxShape: CircleBorder(),
+                        tileColor: Colors.white,
+                        controlAffinity: ListTileControlAffinity.leading,
+                        title: Text(tasks[index].text),
+                        value: tasks[index].isDone,
+                        onChanged: (bool? value) {
+                          setState(() {
+                            tasks[index].isDone = !tasks[index].isDone;
+                          });
+                        },
+                        secondary: IconButton(
+                          iconSize: 30,
+                          color: Colors.amber,
+                          isSelected: tasks[index].isFavorite,
+                          icon: const Icon(
+                            Icons.star_border,
+                          ),
+                          selectedIcon: const Icon(
+                            Icons.star,
+                          ),
+                          onPressed: () {
+                            setState(() {
+                              tasks[index].isFavorite = !tasks[index].isFavorite;
+                            });
+                          },
+                        ))));
+          });
+    } else {
+      return Center(
+          child: Column(
+        mainAxisAlignment: MainAxisAlignment.center,
+        children: [
+          Stack(
+            children: [
+              SvgPicture.asset('assets/todolist_background.svg'),
+              SvgPicture.asset('assets/todolist.svg'),
+            ],
+          ),
+          const Text(
+            'На данный момент задачи отсутствуют',
+            style: TextStyle(fontSize: 20),
+          ),
+        ],
+      ));
```

Comment:
```suggestion
      ),);
```
И отформатировать, чтобы посимпатичнее смотрелось